app/lib/features/attachments/actions/handle_selected_attachments.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:io';
```

Comment:
No changes in code expect `link` type attachment support and putted this as separate function in `Actions` folder.